src/backend/routes/views/account/get/confirmPasswordReset.js

```diff
@@ -1,16 +1,40 @@
 exports = module.exports = function (req, res) {
     const locals = res.locals
 
+    const overallRes = res
```

Comment:
is this a express best practice? or did i miss something?

Reply:
Unsure about best practice, but it is how flashes on redirect are done in many other places in the code base.

Reply:
we lost a lot of history due my refactoring, but most of the code is old and was just a copy

Reply:
Ran into a lot of issues with that syntax for passing the flash to the redirect:

`overallRes.render('account/confirmResyncAccount', { flash })`

Will see if there's a better way to do this and potentially adapt it in other places.

Reply:
You are correct here - overallres was bad practice, just a remnant while copying over using params for flash.